VKApp/VKApp/Services/VKService.swift

```diff
@@ -0,0 +1,38 @@
+// VKService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Alamofire
+import Foundation
+
+/// Менеджер сетевых запросов по API VK
+final class VKService {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let methodText = "/method/"
+        static let fieldsText = "fields"
+        static let accessTokenText = "access_token"
+        static let vText = "v"
+        static let bdateText = "bdate"
+        static let bdateNumberText = "5.131"
+        static let baseUrl = "https://api.vk.com"
+    }
+
+    // MARK: - Public Methods
+
+    func loadVKData(method: String, parameterMap: [String: String]) {
+        let path = Constants.methodText + method
```

Comment:
```suggestion
        let path = 
```
меняем на интерполяцию